analyzers/src/ITs.JsonParser/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using CommandLine;
+using ITs.JsonParser;
+
+Parser.Default.ParseArguments<CommandLineOptions>(args).MapResult(Execute, errs => 1);
+
+int Execute(CommandLineOptions options)
+{
+    WriteLineColor("Processing analyzer results", ConsoleColor.Yellow);
+    NewIssueReports();
+    ShowDiffResults(options.Project, options.RuleId);
```

Comment:
For now I left the same names that the powershell functions have, this will be reworked in the next 2-3 PRs.

Reply:
I'd prefer to go clean already